src/services/concepts.service.ts

```diff
@@ -0,0 +1,37 @@
+import {Observable} from 'rxjs/Rx';
+import {AllOfUsConfig} from '../config';
+import {Concept, ConceptsApi, SearchConceptsRequest} from '../generated/api';
+import {AuthService} from './auth.service';
+
+export class ConceptsService {
+
+  private authService: AuthService;
+  private apiHost: string;
+  private workspaceNamespace: string;
+  private workspaceId: string;
+
+  constructor(authService: AuthService,
+              configObservable: Observable<AllOfUsConfig>) {
+    this.authService = authService;
+    configObservable.subscribe((config) => {
+      this.apiHost = config.apiHost;
+      this.workspaceNamespace = config.workspaceNamespace;
+      this.workspaceId = config.workspaceId;
+    });
+  }
+
+  private conceptsApi(): ConceptsApi {
+    return new ConceptsApi({ basePath: 'https://' + this.apiHost,
```

Comment:
nit: clean up the object formatting, e.g.:

```
new ConceptsApi({
  basePath: ...,
  accessToken: ...
})
```

Reply:
Done.